rabbitmq/datadog_checks/rabbitmq/openmetrics/metrics.py

```diff
@@ -0,0 +1,228 @@
+    "rabbitmq_queue_disk_reads_total": "queue.disk_reads",
+    "rabbitmq_queue_disk_writes_total": "queue.disk_writes",
+    "rabbitmq_queue_head_message_timestamp": "queue.head_message_timestamp",
+    "rabbitmq_queue_index_read_ops_total": "queue.index.read_ops",
+    "rabbitmq_queue_index_write_ops_total": "queue.index.write_ops",
+    "rabbitmq_queue_messages": "queue.messages",
+    "rabbitmq_queue_messages_bytes": "queue.messages.bytes",
+    "rabbitmq_queue_messages_paged_out": "queue.messages.paged_out",
+    "rabbitmq_queue_messages_paged_out_bytes": "queue.messages.paged_out_bytes",
+    "rabbitmq_queue_messages_persistent": "queue.messages.persistent",
+    "rabbitmq_queue_messages_persistent_bytes": "queue.messages.persistent_bytes",
+    "rabbitmq_queue_messages_published_total": "queue.messages.published",
+    "rabbitmq_queue_messages_ram": "queue.messages.ram",
+    "rabbitmq_queue_messages_ram_bytes": "queue.messages.ram_bytes",
+    "rabbitmq_queue_messages_ready": "queue.messages.ready",
+    "rabbitmq_queue_messages_ready_bytes": "queue.messages.ready_bytes",
+    "rabbitmq_queue_messages_ready_ram": "queue.messages.ready_ram",
+    "rabbitmq_queue_messages_unacked": "queue.messages.unacked",
+    "rabbitmq_queue_messages_unacked_bytes": "queue.messages.unacked_bytes",
+    "rabbitmq_queue_messages_unacked_ram": "queue.messages.unacked_ram",
+    "rabbitmq_queue_process_memory_bytes": "queue.process.memory_bytes",
+    "rabbitmq_queue_process_reductions_total": "queue.process.reductions",
+    "rabbitmq_queues": "queues",
+    "rabbitmq_queues_created_total": "queues.created",
+    "rabbitmq_queues_declared_total": "queues.declared",
+    "rabbitmq_queues_deleted_total": "queues.deleted",
+    "rabbitmq_raft_entry_commit_latency_seconds": "raft.entry_commit_latency_seconds",
+    "rabbitmq_raft_log_commit_index": "raft.log_commit_index",
+    "rabbitmq_raft_log_last_applied_index": "raft.log_last_applied_index",
+    "rabbitmq_raft_log_last_written_index": "raft.log_last_written_index",
+    "rabbitmq_raft_log_snapshot_index": "raft.log_snapshot_index",
+    "rabbitmq_raft_term_total": "raft.term",
+    "rabbitmq_resident_memory_limit_bytes": "resident_memory_limit_bytes",
+    "rabbitmq_schema_db_disk_tx_total": "schema_db.disk_tx",
+    "rabbitmq_schema_db_ram_tx_total": "schema_db.ram_tx",
+    "telemetry_scrape_duration_seconds": "telemetry.scrape.duration_seconds",
+    "telemetry_scrape_encoded_size_bytes": "telemetry.scrape.encoded_bytes",
+    "telemetry_scrape_size_bytes": "telemetry.scrape.bytes",
+}
+
+UNTYPED_METRICS = [
+    "erlang_vm_smp_support",
+    "erlang_vm_threads",
+    "erlang_vm_time_correction",
+    "rabbitmq_alarms_file_descriptor_limit",
+    "rabbitmq_alarms_free_disk_space_watermark",
+    "rabbitmq_alarms_memory_used_watermark",
+    "rabbitmq_build_info",
+    "rabbitmq_identity_info",
+]
+METRICS_RENAMES = {k: v for k, v in _METRIC_RENAMES.items() if k not in UNTYPED_METRICS}
```

Comment:
We simply ignore untyped metrics until we decide what to do with them. This discussion is in progress and will definitely be addressed in a follow-up PR.

Reply:
Indeed some of these metrics will be turned into gauges. The `build_info` and `identity_info` will be handled the same way we handle similar metrics in the cockroachdb integration.

Thanks for the amazon_msk link, Fanny and I were searching for that earlier today :)